### PR TITLE
perf: cli support deploy aad manifest

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

**Review by Alive-Fish — DISMISSED** [review]

File: packages/fx-core/src/component/question.ts [hunk 1]
```diff
@@ -216,7 +216,14 @@ export async function getQuestionsForDeployV3(
...
       default: "no",
     });
+    const aadManifestFilePathNode = new QTreeNode({
+      name: Constants.AAD_MANIFEST_FILE,
+      type: "singleFile",
+      title: getLocalizedString("core.aad.aadManifestFilePath"),
+      default: "",
+    });
     node.addChild(aadNode);
+    node.addChild(aadManifestFilePathNode);
```
Comment (KennethBWSong) [inline packages/fx-core/src/component/question.ts:226]: May need to skip this question for vsc?
You can check condition key for QtreeNode to support this. For detail you can reach out for huajie

Comment (wenytang-ms) [reply packages/fx-core/src/component/question.ts]: thanks, code updated.

File: packages/cli/src/telemetry/cliTelemetryEvents.ts [hunk 2]
```diff
@@ -45,6 +45,7 @@ export enum TelemetryEvent {
 
   DeployStart = "deploy-start",
   Deploy = "deploy",
+  DeployAad = "deployAad",
```
Comment (KennethBWSong) [inline packages/cli/src/telemetry/cliTelemetryEvents.ts:48]: Note: Please make sure correctly tag the property in cli telemetry.

**Review by Alive-Fish — APPROVED** [review]

**Review by KennethBWSong — APPROVED** [review]

File: packages/cli/src/telemetry/cliTelemetryEvents.ts (hunk 2 above)
Comment (wenytang-ms) [reply packages/cli/src/telemetry/cliTelemetryEvents.ts:48]: sure, will check it after rc.

**Review by SLdragon — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**